src/model/json-schema.json

```diff
@@ -2741,6 +2741,9 @@
                 "brazeApiKey": {
                     "type": "string"
                 },
+                "ipsosTag": {
+                    "type": "string"
+                },
```

Comment:
Not sure why this crept in…?

Reply:
if you change the TS types, we auto generate a schema based on those types to use to validate CAPI

Reply:
No sure where the TS type would have changed, in this case.

Reply:
I think it was @jfsoul who forgot to push it as part of his PR https://github.com/guardian/dotcom-rendering/pull/2396

I think its fine